networking/hardware_networks/about-sriov.adoc

```diff
@@ -19,49 +29,32 @@ SR-IOV Network Operator. The SR-IOV Network Operator creates and manages the
 components of the SR-IOV stack. The Operator provisions the following
 components:
 
-* Provision the SR-IOV network operator deployment on master nodes.
-* Provision the SR-IOV network config daemon on worker nodes.
-* Provision the Operator webhook on master nodes.
-* Provision the Network resources injector on master nodes.
-* Provision the SR-IOV network device plug-in on worker nodes.
-* Provision the SR-IOV CNI plug-in executable on worker nodes.
-
-Here's the function of each above mentioned SR-IOV components.
-
-* The SR-IOV Operator is a Kubernetes Deployment that manages all SR-IOV
-components in a cluster. It watches creation, update and deletion of Operator
-Custom Resources and takes corresponding actions such as generating
-NetworkAttachmentDefinition Custom Resources for SR-IOV CNI, creating and
-updating configuration of SR-IOV network device plug-in, creating node
-specific SriovNetworkNodeState Custom Resources and updating Spec.Interfaces
-field in each SriovNetworkNodeState Custom Resource, etc.
-
-* The SR-IOV network config daemon is a Kubernetes DaemonSet deployed on
-worker nodes when SR-IOV Operator is launched. It is responsible for
-discovering and initializing SR-IOV network devices in cluster.
-
-* The Operator webhook is a Kubernetes Dynamic Admission Controller Webhook
-that validates correctness of Operator Custom Resource and sets default values
-for fields in Operator Custom Resource that are not configured by user.
-
-* The Network resources injector is a Kubernetes Dynamic Admission Controller
-Webhook that provides functionality for patching Kubernetes Pod specifications
-with requests and limits for custom network resources such as SR-IOV VFs.
-
-* The SR-IOV network device plug-in is a Kubernetes device plug-in for
-discovering, advertising, and allocating SR-IOV network virtual function (VF)
-resources. Device plug-ins are used in Kubernetes to enable the use of limited
-resources, typically in physical devices. Device plug-ins give the Kubernetes
-scheduler awareness of resource availability, so the scheduler can schedule
-Pods on nodes with sufficient resources.
-
-* The SR-IOV CNI plug-in plumbs VF interfaces allocated from the SR-IOV device
-plug-in directly into a Pod.
+SR-IOV Network Operator::
+A controller that orchestrates discovery and management of SR-IOV network devices:
```

Comment:
I don't think that it's appropriate to end in a colon as is. I was expecting a list of network devices to follow the way it is worded now. I'd suggest adding something like it performs the following functions or something.